c7n/query.py

```diff
@@ -328,8 +328,8 @@ def get_query_params(self, query):
         else:
             _c = None
 
-        s = "select configuration, supplementaryConfiguration where resourceType = '{}'".format(
-            self.manager.resource_type.config_type)
+        s = ("select resourceId, configuration, supplementaryConfiguration "
```

Comment:
this seems unrelated to the pagination on sqs and also seems unused, could you clarify what the intent is ?